onnx/backend/test/case/node/meansquaredistance.py

```diff
@@ -0,0 +1,132 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import numpy as np  # type: ignore
+
+import onnx
+from ..base import Base
+from . import expect
+
+
+class MeanSquaredDistance(Base):
```

Comment:
It would be good to have a reference implementation function.
As explained in the comment in the deleted file : https://github.com/onnx/onnx/pull/2570#discussion_r372130505